app/views/measures/_measures.html.erb

```diff
@@ -144,6 +146,29 @@
     <%= render 'headings/footnote', footnotes: declarable.footnotes %>
   </div>
 
+  <!-- Footnotes tab -->
+  <div class="govuk-tabs__panel" id="footnotes">
+    <h2 class="govuk-heading-m"><%= footnote_heading(declarable) %></h2>
+    <table class="govuk-table govuk-!-margin-top-8">
+      <colgroup>
+        <col width="100">
+        <col width="*">
+      </colgroup>
+      <thead class="govuk-table__head">
+        <tr class="govuk-table__row">
+          <th scope="col" class="govuk-table__header">Code</th>
+          <th scope="col" class="govuk-table__header">Description</th>
+        </tr>
+      </thead>
+      <tbody class="govuk-table__body">
+        <% declarable.footnotes.each do |footnote| %>
+          <% next unless footnote&.code.present? %>
+          <%= render footnote %>
+        <% end %>
```

Comment:
You can pass `declarable.footnotes` as a collection to `footnote` partial and you won't need this loop this way. We can chat once you're back.